prac_07/convert_miles_km/convert_miles_km.py

```diff
@@ -0,0 +1,37 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KM = 1.60934
+
+
+def get_float(text: str):
```

Comment:
I think this line can be moved into the class itself, and maybe give it a better name like validate_input or something?